src/components/Info.jsx

```diff
@@ -0,0 +1,19 @@
+import "./Info.css";
+const Info = ({ folderUser }) => {
+  if (!folderUser || !folderUser.folder) {
+    return null;
+  }
+  const { name } = folderUser.folder;
+  if (!folderUser.folder.owner) {
+    return null;
+  }
+  const { name: ownerName, profileImageSource } = folderUser.folder.owner;
```

Comment:
위에 말씀드렸듯 중간에 return 을 하는 것보다는 아래처럼 해보면 어떨까합니다!

```suggestion
  const { name, owner } = folderUser.folder
  const { name: ownerName, profileImageSource } = owner
  
  const isUserInfoExist = name && ownerName && profileImageSource
  
  return (
     isUserInfoExist && {
       <div className="Info">
      <img src={profileImageSource} alt="profileImageSource" />
      <p>@{ownerName}</p>
      <h1>{name}</h1>
    </div>
     }
  )
  
```